Add documentation for using multiple drop-shadows (#37138)
mdn/content: files/en-us/web/css/filter-function/drop-shadow/index.md

/* The order of color and length values can be changed */
/* drop-shadow( <color> <length> <length> <length> ) */
drop-shadow(#e23 0.5rem 0.5rem 1rem)

> /* You can pass multiple drop-shadow's to a filter to stack them */

Reviewer: ```suggestion
/* Pass multiple drop-shadows to a filter to stack them */
```

<div>drop-shadow(16px 16px red)</div>
<div>drop-shadow(red 1rem 1rem 10px)</div>
<div>drop-shadow(-16px -16px red)</div>
<div>
  drop-shadow(1px 1px red)
  drop-shadow(1px -1px red)

>   drop-shadow(-1px 1px red)

Author: These should either all be on the same line, or all be on separate lines.

* All same lines conveys that it is all part of the same CSS property/value.
* All on separate lines makes the repetition easier to visually parse.

All the suggestions the linter is making are bad.

>   drop-shadow(-1px 1px red)

Reviewer: Hi @TheJaredWilcurt. Yup, the linter is really annoying sometimes. To stop it from committing linting crimes in cases like this, update the language specifier on the opening code fence to include `-nolint`, so for example

<pre>```css-nolint
/* CSS code here */
```</pre>

or

<pre>```js-nolint
// JS code here
```</pre>

>   drop-shadow(-1px 1px red)

Reviewer: Then re add, commit, and push, and it should turn the PR green.

<div>drop-shadow(16px 16px red)</div>
<div>drop-shadow(red 1rem 1rem 10px)</div>
<div>drop-shadow(-16px -16px red)</div>

> <div>drop-shadow(1px 1px red) drop-shadow(1px -1px red) drop-shadow(-1px 1px red) drop-shadow(-1px -1px red)</div>

Author: ```suggestion
<div>
  drop-shadow(1px 1px red)
  drop-shadow(1px -1px red)
  drop-shadow(-1px 1px red)
  drop-shadow(-1px -1px red)
</div>
```

div:nth-child(4) {
  filter: drop-shadow(-16px -16px red);
}

div:nth-child(5) {

>   filter:
>     drop-shadow(1px 1px red)
>     drop-shadow(1px -1px red)
>     drop-shadow(-1px 1px red)
>     drop-shadow(-1px -1px red);

Author: ```suggestion
  filter: drop-shadow(1px 1px red)
          drop-shadow(1px -1px red)
          drop-shadow(-1px 1px red)
          drop-shadow(-1px -1px red);
```

div:nth-child(4) {
  filter: drop-shadow(-16px -16px red);
}

div:nth-child(5) {
  filter: drop-shadow(1px 1px red)
          drop-shadow(1px -1px red)
          drop-shadow(-1px 1px red)

>           drop-shadow(-1px -1px red);

Author: Is there a way to make the linter shut up without making the code look like it was written by AI?

div:nth-child(4) {
  filter: drop-shadow(-16px -16px red);
}

div:nth-child(5) {

>   filter: drop-shadow(1px 1px red)
>           drop-shadow(1px -1px red)
>           drop-shadow(-1px 1px red)
>           drop-shadow(-1px -1px red);

Author: ```suggestion
  filter:
    drop-shadow(1px 1px red)
    drop-shadow(1px -1px red)
    drop-shadow(-1px 1px red)
    drop-shadow(-1px -1px red);
```

div:nth-child(4) {
  filter: drop-shadow(-16px -16px red);
}

div:nth-child(5) {

>   filter:

Reviewer: My only comment here is that I would make the four shadows a bit more different, in color and/or placement. At the moment, it is difficult to see that multiuple shadows are being applied, as they are all so close together.

Author: That's fair. This 1/1, 1/-1, -1/1, -1/-1 approach is a hack people use to add a "stroke" to elements/text. It's the most common multi-shadow I've used over the years.

Reviewer: Cool, that makes sense.

    drop-shadow(1px 1px red)
    drop-shadow(1px -1px red)
    drop-shadow(-1px 1px red)
    drop-shadow(-1px -1px red);
}
```

> {{EmbedLiveSample("Setting a drop shadow", "100%", "300px")}}

Reviewer: You also need to update the paragraph following the example to describe your new addition. At the moment, it only describes the first four shadows.
